landowner.h

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <vector>
+
+class Landowner
+/**/
+{
+private:
```

Comment:
https://cb-geo.github.io/cpp-guide/#/ for naming and https://www.doxygen.nl/manual/docblocks.html we follow 3b style option